test/parallel.jl

```diff
@@ -61,3 +64,5 @@ winners = mapToGigaSOM(som2, dfSom)
     end
 
 end
+
+rmprocs(workers());
```

Comment:
No need for a `;`

Reply:
I also found it weird, but it was writtien like this on slack ;) ill remove it